src/c/project-card.js

```diff
@@ -0,0 +1,56 @@
+window.c.ProjectCard = (function(m, h, models){
+  return {
+    controller: function(args) {
+      var project = args.project;
+      return {
+        remainingTextObj: h.generateRemaingTime(project)
```

Comment:
This function call should be in the view, in case we update the data it can be recalculated.
The ProjectCard won't need a controller :D